client/coral-embed-stream/src/Embed.js

```diff
@@ -158,7 +158,7 @@ class Embed extends Component {
                 this.props.data.refetch();
               }}>{lang.t('showAllComments')}</Button>
           }
-          {loggedIn && <UserBox user={user} logout={() => this.props.logout().then(refetch)}  changeTab={this.changeTab}/>}
+          {loggedIn && activeTab !== 1 && <UserBox user={user} logout={() => this.props.logout().then(refetch)}  changeTab={this.changeTab}/>}
```

Comment:
Oh sorry, this wasn't written clearly, @okbel, we should remove the username, not the Logged in message.

<img width="557" alt="username" src="https://cloud.githubusercontent.com/assets/1077300/24937940/1692343c-1f02-11e7-9ae8-e165d57ea75c.png">